capstone/src/main/webapp/text_analyzer.html

```diff
@@ -59,25 +59,36 @@ <h1>Text Analysis</h1>
             <option value="it">Italian</option>
             <option value="pt">Portuguese</option>
           </select>
+
           <div class="dropdown" id="attribute-dropdown">
             <button class="btn btn-secondary dropdown-toggle" type="button" data-toggle="dropdown">Attributes</button>
             <ul class="dropdown-menu checkbox-menu allow-focus" id="available-attributes"></ul>
           </div>
-          <div id="analysis-type">
-            <b>Select analysis type</b>
-            <div class="form-check form-check-inline">
-              <input class="form-check-input" type="radio" name="analysisRadios" id="bySentence" value="SENTENCE" checked>
-              <label class="form-check-label" for="bySentence">By sentence</label>
-            </div>
-            <div class="form-check form-check-inline">
-              <input class="form-check-input" type="radio" name="analysisRadios" id="byWord" value="WORD">
-              <label class="form-check-label" for="byWord">By word</label>
-            </div>
-            <div class="form-check form-check-inline">
-              <input class="form-check-input" type="radio" name="analysisRadios" id="noAnalysis" value="NONE">
-              <label class="form-check-label" for="noAnalysis">No analysis</label>
-            </div>
+
```

Comment:
nit: Let's remove this extra line and the one on line 91

Reply:
Oops, I forgot about this. Fixed!